### PR TITLE
linux: move to 4.8-rc8 with no patches for POWER9 platforms

### PR DESCRIPTION
**Pull request by stewart-ibm**
Instead of attempting to apply teh 4.4.x openpower patches, apply
no patches for POWER9 systems (for now).

Fixes: https://github.com/open-power/op-build/issues/659
Signed-off-by: Stewart Smith <stewart@linux.vnet.ibm.com>

---
This change is [Reviewable](https://reviewable.io/reviews/open-power/op-build/661)

### CODE REVIEW

**Comment by stewart-ibm** [comment]
@jazurin broken build slave.

**Comment by jazurin** [comment]
ok @stewart-ibm,  I added the missing linux-4.8-rc8.tar.xz.  Btw IT is telling me that wget is scheduled to be updated by the end of the month, that should fix these download issues.

retest this please

File: openpower/configs/witherspoon_defconfig [hunk 1]
```diff
@@ -42,7 +42,8 @@ BR2_ROOTFS_DEVICE_TABLE="../openpower/device_table.txt"
 BR2_ROOTFS_POST_BUILD_SCRIPT="../openpower/scripts/fixup-target-var"
 BR2_LINUX_KERNEL=y
 BR2_LINUX_KERNEL_CUSTOM_TARBALL=y
-BR2_LINUX_KERNEL_CUSTOM_TARBALL_LOCATION="https://cdn.kernel.org/pub/linux/kernel/v4.x/testing/linux-4.8-rc5.tar.xz"
+BR2_LINUX_KERNEL_CUSTOM_TARBALL_LOCATION="https://cdn.kernel.org/pub/linux/kernel/v4.x/testing/linux-4.8-rc8.tar.xz"
+BR2_LINUX_KERNEL_PATCH=""
 BR2_KERNEL_HEADERS_4_4=y
 BR2_BINUTILS_VERSION_2_26_X=y
 BR2_LINUX_KERNEL_PATCH="$(BR2_EXTERNAL)/linux"
```
Comment (jazurin) [inline openpower/configs/witherspoon_defconfig]: @stewart-ibm Shouldn't this line be deleted?  I built a witherspoon defconfig and it still tried to apply patches and failed.
```
WARNING: no hash file for linux-4.8-rc8.tar.xz
>>> linux custom Extracting
xzcat /tmp/hostboot/jenkins-slave/workspace/op_build_gerrit_trigger_p9_multi/CONFIG/witherspoon/OS/rhel6.4/build/op-build/dl/linux-4.8-rc8.tar.xz | tar --strip-components=1 -C /tmp/hostboot/jenkins-slave/workspace/op_build_gerrit_trigger_p9_multi/CONFIG/witherspoon/OS/rhel6.4/build/op-build/output/build/linux-custom   -xf -
>>> linux custom Patching
for p in /tmp/hostboot/jenkins-slave/workspace/op_build_gerrit_trigger_p9_multi/CONFIG/witherspoon/OS/rhel6.4/build/op-build/openpower/linux ; do if test -d $p ; then support/scripts/apply-patches.sh  /tmp/hostboot/jenkins-slave/workspace/op_build_gerrit_trigger_p9_multi/CONFIG/witherspoon/OS/rhel6.4/build/op-build/output/build/linux-custom $p \*.patch || exit 1 ; else support/scripts/apply-patches.sh  /tmp/hostboot/jenkins-slave/workspace/op_build_gerrit_trigger_p9_multi/CONFIG/witherspoon/OS/rhel6.4/build/op-build/output/build/linux-custom `dirname $p` `basename $p` || exit 1; fi done

Applying 0001-xhci-Use-xhci_pci_remove-for-xhci-device-shutdown.patch using patch: 
patching file drivers/usb/host/xhci-pci.c
Hunk #1 succeeded at 462 (offset 15 lines).

Applying 0002-drivers-drm-ast-Switch-SCU-to-VGA-output-on-POST.patch using patch: 
patching file drivers/gpu/drm/ast/ast_post.c

Applying 0003-net-mlx4_core-Set-UAR-page-size-to-4KB-regardless-of.patch using patch: 
patching file drivers/infiniband/hw/mlx4/qp.c
Reversed (or previously applied) patch detected!  Skipping patch.
1 out of 1 hunk ignored -- saving rejects to file drivers/infiniband/hw/mlx4/qp.c.rej
patching file drivers/net/ethernet/mellanox/mlx4/cq.c
Reversed (or previously applied) patch detected!  Skipping patch.
1 out of 1 hunk ignored -- saving rejects to file drivers/net/ethernet/mellanox/mlx4/cq.c.rej
patching file drivers/net/ethernet/mellanox/mlx4/en_resources.c
Reversed (or previously applied) patch detected!  Skipping patch.
1 out of 1 hunk ignored -- saving rejects to file drivers/net/ethernet/mellanox/mlx4/en_resources.c.rej
patching file drivers/net/ethernet/mellanox/mlx4/en_tx.c
Reversed (or previously applied) patch detected!  Skipping patch.
1 out of 1 hunk ignored -- saving rejects to file drivers/net/ethernet/mellanox/mlx4/en_tx.c.rej
patching file drivers/net/ethernet/mellanox/mlx4/eq.c
Reversed (or previously applied) patch detected!  Skipping patch.
1 out of 1 hunk ignored -- saving rejects to file drivers/net/ethernet/mellanox/mlx4/eq.c.rej
patching file drivers/net/ethernet/mellanox/mlx4/main.c
Reversed (or previously applied) patch detected!  Skipping patch.
5 out of 5 hunks ignored -- saving rejects to file drivers/net/ethernet/mellanox/mlx4/main.c.rej
patching file drivers/net/ethernet/mellanox/mlx4/pd.c
Reversed (or previously applied) patch detected!  Skipping patch.
1 out of 1 hunk ignored -- saving rejects to file drivers/net/ethernet/mellanox/mlx4/pd.c.rej
patching file include/linux/mlx4/device.h
Reversed (or previously applied) patch detected!  Skipping patch.
3 out of 3 hunks ignored -- saving rejects to file include/linux/mlx4/device.h.rej
make[1]: *** [/tmp/hostboot/jenkins-slave/workspace/op_build_gerrit_trigger_p9_multi/CONFIG/witherspoon/OS/rhel6.4/build/op-build/output/build/linux-custom/.stamp_patched] Error 1
make: *** [_all] Error 2
make: Leaving directory `/tmp/hostboot/jenkins-slave/workspace/op_build_gerrit_trigger_p9_multi/CONFIG/witherspoon/OS/rhel6.4/build/op-build/buildroot'
+ '[' 2 -ne 0 ']'
```

Comment (stewart-ibm) [inline openpower/configs/witherspoon_defconfig]: Yes, you're correct. Missed it. Patch incoming.

**Pull request merged**